From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Mon, 10 Jul 2023 22:05:08 +0200
Subject: [PATCH] Material fixes - Add missing value to isTransparent

Adds the missing legacy value `LEGACY_BARRIER` to `Material.isTransparent`.

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Fiddle - https://fiddlemc.org

diff --git a/src/main/java/org/bukkit/Material.java b/src/main/java/org/bukkit/Material.java
index 9d64d46533ba18470210535294e88cee32c18178..27368dc623859eade5a01abe3033a5ff225e7f79 100644
--- a/src/main/java/org/bukkit/Material.java
+++ b/src/main/java/org/bukkit/Material.java
@@ -7412,6 +7412,7 @@ public enum Material implements Keyed, Translatable, net.kyori.adventure.transla
             case LEGACY_REDSTONE_COMPARATOR_OFF:
             case LEGACY_REDSTONE_COMPARATOR_ON:
             case LEGACY_ACTIVATOR_RAIL:
+            case LEGACY_BARRIER: // Fiddle - add missing Material values that are transparent
             case LEGACY_CARPET:
             case LEGACY_DOUBLE_PLANT:
             case LEGACY_END_ROD:
